eclair-core/src/test/scala/fr/acinq/eclair/payment/PaymentPacketSpec.scala

```diff
@@ -382,21 +382,21 @@ class PaymentPacketSpec extends AnyFunSuite with BeforeAndAfterAll {
     assert(payload_e == FinalPayload.Standard(TlvStream(AmountToForward(finalAmount), OutgoingCltv(finalExpiry), PaymentData(invoice.paymentSecret, finalAmount), OnionPaymentPayloadTlv.PaymentMetadata(hex"010203"))))
   }
 
-  test("fail to build outgoing trampoline payment when too much invoice data is provided") {
+  test("build outgoing trampoline payment with larger trampoline onion size when a large invoice data is provided") {
```

Comment:
We can improve those tests by verifying that the payloads can be decrypted all the way to the recipient.